lib/plug/crypto/message_verifier.ex

```diff
@@ -8,11 +8,13 @@ defmodule Plug.Crypto.MessageVerifier do
   tampered with.
   """
 
+  @delimiter "++"
```

Comment:
I don't mind that. Currently, it looks like url_base64 alphabet is used for encoding which doesn't include `+`.